Description: Make 20-sane.hwdb Multi-Arch ready
Author: Jörg Frings-Fürst <debian@jff.email>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=880391
Forwarded: not-needed
Last-Update: 2018-04-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/tools/sane-desc.c
===================================================================
--- trunk.orig/tools/sane-desc.c
+++ trunk/tools/sane-desc.c
@@ -3326,10 +3326,10 @@ print_usermap_header (void)
 
   printf
     ("# This file was automatically created based on description files (*.desc)\n"
-    "# by sane-desc %s from %s on %s"
+    "# by sane-desc %s from %s\n"
     "#\n"
     ,
-    SANE_DESC_VERSION, PACKAGE_STRING, asctime (localtime (&current_time)));
+    SANE_DESC_VERSION, PACKAGE_STRING);
 
   printf
      ("# The entries below are used to detect a USB device and change owner\n"
Index: trunk/testsuite/tools/data/db.ref
===================================================================
--- trunk.orig/testsuite/tools/data/db.ref
+++ trunk/testsuite/tools/data/db.ref
@@ -1,5 +1,5 @@
 # This file was automatically created based on description files (*.desc)
-# by sane-desc 3.5 from sane-backends 1.0.24git on Wed Jul 31 07:52:48 2013
+# by sane-desc 3.5 from sane-backends 1.0.24git
 #
 # The entries below are used to detect a USB device when it's plugged in
 # and then run a script to change the ownership and
Index: trunk/testsuite/tools/data/hwdb.ref
===================================================================
--- trunk.orig/testsuite/tools/data/hwdb.ref
+++ trunk/testsuite/tools/data/hwdb.ref
@@ -1,5 +1,5 @@
 # This file was automatically created based on description files (*.desc)
-# by sane-desc 3.5 from sane-backends 1.0.25git on Tue Dec  3 15:24:46 2013
+# by sane-desc 3.5 from sane-backends 1.0.25git
 #
 # hwdb file for supported USB devices
 #
Index: trunk/testsuite/tools/data/udev+acl.ref
===================================================================
--- trunk.orig/testsuite/tools/data/udev+acl.ref
+++ trunk/testsuite/tools/data/udev+acl.ref
@@ -1,5 +1,5 @@
 # This file was automatically created based on description files (*.desc)
-# by sane-desc 3.5 from sane-backends 1.0.24git on Wed Jul 31 07:52:49 2013
+# by sane-desc 3.5 from sane-backends 1.0.24git
 #
 # udev rules file for supported USB and SCSI devices
 #
Index: trunk/testsuite/tools/data/udev+hwdb.ref
===================================================================
--- trunk.orig/testsuite/tools/data/udev+hwdb.ref
+++ trunk/testsuite/tools/data/udev+hwdb.ref
@@ -1,5 +1,5 @@
 # This file was automatically created based on description files (*.desc)
-# by sane-desc 3.5 from sane-backends 1.0.24git on Thu Aug  1 18:50:15 2013
+# by sane-desc 3.5 from sane-backends 1.0.24git
 #
 # udev rules file for supported USB and SCSI devices
 #
Index: trunk/testsuite/tools/data/udev.ref
===================================================================
--- trunk.orig/testsuite/tools/data/udev.ref
+++ trunk/testsuite/tools/data/udev.ref
@@ -1,5 +1,5 @@
 # This file was automatically created based on description files (*.desc)
-# by sane-desc 3.5 from sane-backends 1.0.24git on Wed Jul 31 07:52:48 2013
+# by sane-desc 3.5 from sane-backends 1.0.24git
 #
 # udev rules file for supported USB and SCSI devices
 #
Index: trunk/testsuite/tools/data/usermap.ref
===================================================================
--- trunk.orig/testsuite/tools/data/usermap.ref
+++ trunk/testsuite/tools/data/usermap.ref
@@ -1,5 +1,5 @@
 # This file was automatically created based on description files (*.desc)
-# by sane-desc 3.5 from sane-backends 1.0.24git on Wed Jul 31 07:52:48 2013
+# by sane-desc 3.5 from sane-backends 1.0.24git
 #
 # The entries below are used to detect a USB device and change owner
 # and permissions on the "device node" used by libusb.
